<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnakeHuClass</class>
 <widget class="QDialog" name="SnakeHuClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>胡婉清和黄达添之决战紫禁之巅</string>
  </property>
  <widget class="QCheckBox" name="cbHuangdatian">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>80</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>huangdatian</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbHuwanqing">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>huwanqing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>分数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>玩家选择</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbScore">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>230</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>ANG</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>270</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>HU</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SnakeHu.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>OnLoadImg()</slot>
  <slot>OnHuangClicked()</slot>
 </slots>
</ui>
